src/generator.rs

```diff
@@ -95,7 +94,6 @@ RemainAfterExit=yes
 ExecStartPre=-modprobe zram
 ExecStart={generator} --setup-device '%i'
 ",
```

Comment:
Apart from the executable path, this unit is now fully generic. So it could be stored as a static file in /usr/lib/systemd/system/. (Units which have a static file are in general less confusing, because people can more easily figure out where they are coming from.)